src/lib/libwasmfs_fetch.js

```diff
@@ -90,6 +90,12 @@ addToLibrary({
       // one request for all the chunks we need, rather than one
       // request per chunk.
       var start = firstChunk * chunkSize;
+
+      // Out of bounds. No request necessary.
+      if (start >= wasmFS$JSMemoryRanges[file].size) {
+        return Promise.resolve();
```

Comment:
I think we can just use `return` here actually.  See https://github.com/emscripten-core/emscripten/pull/24443